conftest.py

```diff
@@ -45,9 +44,20 @@ def test_obj(base_url, browser, browser_version, os_version, os_name, remote_fla
             test_obj.set_rp_logger(reportportal_service)
 
         yield test_obj
+
         #Teardown
-        test_obj.wait(3)
-        test_obj.teardown()
+        def fin():
```

Comment:
Do we need to add this finalizer?
Can we not use a `finally` block in `try-catch`?